### PR TITLE
feat(Preprocessing): include pre-extracted text and image embeddings optionally

### PR DESCRIPTION
**Pull request by itchingpixels**

### CODE REVIEW

File: run_preprocessing.py [hunk 1]
```diff
@@ -176,17 +151,58 @@ def preprocess(config: PreprocessingConfig):
...
+            "data/derived/fashion-recommendation-text-embeddings-clip-ViT-B-32.pt"
+        )
+
+        # for key in ["derived_name", "derived_look", "derived_category"]:
+        for index, article in tqdm(articles.iterrows()):
+            per_article_text_embedding = torch.cat(
+                (
+                    per_article_text_embedding,
+                    text_embeddings[int(article["article_id"])]
+                    .get("derived_look", torch.zeros(512))
```
Comment (daralthus) [inline run_preprocessing.py:184]: config?

**Review by daralthus — CHANGES_REQUESTED** [review]

File: run_preprocessing.py (hunk 1 above)
Comment (itchingpixels) [reply run_preprocessing.py:184]: config sounds good, I'll make it happen!

**Review by daralthus — APPROVED** [review]
Would prob put in config to use the embeddings for all of: "derived_name", "derived_look", "derived_category"

**Comment by itchingpixels** [comment]
@daralthus : happy to, but let's see whether these huge embeddings are that useful first. they increase the size of our data by 100x, which may be way too much.... I'd want to merge this, but of course feel free to work on any part you'd like to extend, including the preprocessing step!

**Pull request merged**